source/Ship.cpp

```diff
@@ -51,6 +51,8 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 using namespace std;
 
 namespace {
+	const bool UNIVERSAL_RAMSCOOP = !Preferences::Has("Disable universal ramscoop");
```

Comment:
A constant setting? Idk

Reply:
The alternative is checking it every frame in DoGeneration(). I would prefer not to add an extra call to Preferences::Has() for every ship every frame. I'd rather just delete the thing (and not add any ramscoop to drives).

Reply:
Hmmm... my problem here is that I can not be changed in game. In this case you can leave it like it is???

Reply:
I would want to add a named method specifically for this to Preferences instead of just getting it from the map.